services/core/java/com/android/server/clipboard/ClipboardService.java

```diff
@@ -429,9 +429,7 @@ private void scheduleAutoClear(@UserIdInt int userId, int intendingUid) {
         }
 
         private long getTimeoutForAutoClear() {
-            return DeviceConfig.getLong(DeviceConfig.NAMESPACE_CLIPBOARD,
-                    PROPERTY_AUTO_CLEAR_TIMEOUT,
-                    DEFAULT_CLIPBOARD_TIMEOUT_MILLIS);
+            return Settings.Global.getLong(mContext.getContentResolver(), Settings.Global.CLIPBOARD_AUTO_CLEAR_TIMEOUT, DEFAULT_CLIPBOARD_TIMEOUT_MILLIS);
```

Comment:
ExtSettings checks should import `import android.ext.settings.ExtSettings;` and do:

`ExtSettings.<key>.get(<context>)`

In this case:

`return ExtSettings.CLIPBOARD_AUTO_CLEAR_TIMEOUT.get(mContext)`

As we're not using Settings.java.